ui/install.slint

```diff
@@ -81,22 +81,8 @@ export component CargoInstallView inherits GridBox {
                     horizontal-alignment: right;
                 }
             }
-            if crate.queued : Rectangle {
-                height: crate_name.preferred_height;
-                Rectangle {
-                    y:0;
-                    width: parent.width / 2;
-                    x: parent.width - self.width - 6px;
-                    height: parent.height - 6px;
```

Comment:
On the screenshot, the progressbar seems to extand more than the text in both direction. I suppose it would look better if they had the same margin.  Maybe you just need to put the ProgressIndicator in a HorizontalBox. 
Or the whole thing in a GridBox with `visible: condition;` instead of `if condition:`